### PR TITLE
US-8-create-method-to-add-phone

### PR DESCRIPTION
**Pull request by TaniaSakun**
## Description:
1. The method to add the phone number to the contact was added
2. The validation method for the phone number was created for the future implementation
3. The birthday service was updated (added messages for the responses)

### CODE REVIEW

File: core/core/models/payload.py [hunk 1]
```diff
@@ -25,7 +25,10 @@ class PhonePayload(BaseModel):
     phone: str
     old_phone: Optional[str] = None
 
-
+    def __init__(self):
+   #     validate_phone_number(self.phone)
+        pass
```
Comment (iMykhailychenko) [inline core/core/models/payload.py:30]: I will double-check, but I guess that pydantic (the library that we used here) has its own way to validate models.

File: core/core/services/phones.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    return set_phone_number(payload, record)
+
+
+def set_phone_number(payload, record: Record) -> Record:
+    if record:
+        if payload.phone not in record.phones:
+            record.phones.add(payload.phone)
+        return record
+    else:
+        return Record(name=payload.name, phones=set([payload.phone]))
```
Comment (iMykhailychenko) [inline core/core/services/phones.py:24]: set already store only unique values, so we don't need to check if `payload.phone not in record.phones` and also if record exist than it is should contain empty set by default in `phones` attribute, so we don't need to check if phones exist too.
it should work like this:
```
def set_phone_number(payload, record: Record) -> Record:
    if record:
        record.phones.add(payload.phone)
        return record
    else:
        return Record(name=payload.name, phones={payload.phone})

File: core/core/models/payload.py (hunk 1 above)
Comment (TaniaSakun) [reply core/core/models/payload.py:30]: Okay. I'll remove this part of the code. If it needed we return to it during the US-19 implementation

File: core/tests/test_validations.py [hunk 3]
```diff
@@ -11,9 +11,9 @@ def test_invalid_phone_number():
 
     # result = controller(
     #     Actions.ADD,
-    #     ContactPayload(name="Joe", phones={"12345678912345"}),
-    # )  # to long
-    # assert result.value == ValidationMessages.PHONE_NUMBER_LENGTH.value
+    #     ContactPayload(name="Joe", phones={"testString123"}),
```
Comment (iMykhailychenko) [inline core/tests/test_validations.py:14]: I guess we should have both of them (to long and unapropriate format)

File: core/core/services/phones.py (hunk 2 above)
Comment (TaniaSakun) [reply core/core/services/phones.py:24]: oh...yeah../that's a set, not a list. Got it😊

File: core/tests/test_validations.py (hunk 3 above)
Comment (TaniaSakun) [reply core/tests/test_validations.py:14]: My mistake during merging, sorry. I'll fix it

**Review by iMykhailychenko — APPROVED** [review]

**Pull request merged**